### ISSUE TITLE
Can spaCy use vectors as training data?

### ISSUE THREAD

**Issue by pranav4838** [explosion/spaCy#4236]
Hello all , 
I am currently working on a multi class text classification project. I have a set of .txt files for which I have created the equivalent vector representation using spaCy. I now want to create labelled data which will be my training data in order to train and create a model . 

The training data should be vectors followed by the labels. 

Can someone please direct me on how to get spaCy to accept vectors. The examples I have seen so far use text data followed by labels as the the training data . Are there any examples available that use vectors followed by labels as the training data .

Thank you !!

Environment
spaCy version: 2.1.8
Platform: Linux-4.15.0-52-generic-x86_64-with-Ubuntu-18.04-bionic
Python version: 3.6.8
Models: en_pytt_bertbaseuncased_lg

**Comment by adrianeboyd** [comment]
Right now spacy is only set up to use features it derives directly from the text when training a text classification model. There are feature requests related to this (#4160), but no chosen solution or implementation in spacy yet. You will probably need to use a more general-purpose machine learning library for your task.

What kinds of features are you interested in including?

**Comment by kbharathsreddy** [comment]
Hello,

I am working on a similar project where I am dividing the BERT pipeline into two parts and saving the output from the first part and feeding that data into the second part at a later time. 
First part has Sentencizer, Word Piecer & token2vec.
Second part has textcat

Can I feed my vectors from token2vec from the first part into textcat of the second part? 

Thank you very much!

**Comment by alejandrojcastaneira** [comment]
**Could use the vectors as output data**?
Normally I would recommend to build this architecture from scratch, I just wan to mention the use case: 

Suppose you have and e-commerce: and you have (class 1 . . . class n) for products and also a single product could belong to multiple classes, suppose you want to classify a new product just by the given product description? The explosion of possible combination it's huge for a typical text classification problem.

But what about if similar to the pretraining in spacy instead of a softmax over all the possible classes we could just predict the vector for each class (obtained by custom made methods or embeddings: doc2vec, starspace, etc) Is just an idea which probably way out of the scope of the library but I just wanted to share this use case with you, 

Best regards.
